x/dex/keeper/multihop_swap.go

```diff
@@ -34,7 +34,7 @@ func (k Keeper) HopsToRouteData(
 		}
 		price, found := k.GetCurrPrice(ctx, tradePairID)
 		if !found {
-			return routeArr, types.ErrInsufficientLiquidity
+			return routeArr, types.ErrLimitPriceNotSatisfied
```

Comment:
maybe different error? it's kinda more about missing liquidity pools at all than limit price not satisfied. (yeah i know probably a rare occurence, but still)

Reply:
Decided to simplify things by having a single error here. From a user's perspective, if there's no liquidity or all of the liquidity is above their limitPrice, it's functionally the same.